Snippets/ASQ/ASQ_7/Configuration/MessageWrapperSerializer.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace ASQ_7.Configuration
+{
+    using NServiceBus;
+
+    public class MessageWrapperSerializer
+    {
+        #region SerializerAndMessageWrapperSerializer
+        public void ConfigureEndpointA()
```

Comment:
Maybe instead of using ConfigureEndpointA/B we should refer to the old/new endpoints? To show how to be backward compatible?